Packs/Slack/Integrations/SlackEventCollector/SlackEventCollector.py

```diff
@@ -120,6 +121,7 @@ def get_logs_with_pagination(self, query_params: dict, last_run: dict) -> List[d
             # we need to know where to stop in the next runs
             last_run['last_id'] = aggregated_logs[0].get('id')
 
+        demisto.debug(f'aggregated_logs: {aggregated_logs}')
```

Comment:
I don't recommend printing all the logs in the debug log, pick a small amount of relevant details (also in other logs)